### DISCUSSION TITLE
PDB2PQR online webserver not working for nucleic acids

### DISCUSSION THREAD

**Discussion by srdjan125** [Electrostatics/apbs#158] [Q&A]
Hello.

I I have tried to create PQR file of nucleic acid (of RNA molecule) by PDB2PQR webserver. I am doing it on same way like I was doing for proteins: upload the pdb file and choose pH and force field. I have tried both CHARMM, PARSE and AMBER force field but webserver is not giving me PQR file, it gives some file but not pqr.
  May I ask what is the problem here? Or how can I make PQR of RNA?

**Comment by srdjan125** [comment]
I have found solution in old emailing list:  residue names need to be changed from URA to RU ADE top RA etc in order for PDB2PQR to recognize RNAs. Please delete this topic, thank you. (I cannot delete it by myself).

**Comment by intendo** [comment]
Thank you for posting your solution. This may help someone else in the future.

**Reply by srdjan125** [reply]
Ok no problem. I have found solution in some old emailing list (after doing google search).
